src/attributes.cpp

```diff
@@ -2539,8 +2539,8 @@ namespace attributes {
             ostr() << "# Register entry points for exported C++ functions"
                    << std::endl;
             ostr() << "methods::setLoadAction(function(ns) {" << std::endl;
-            ostr() << "    .Call('" << registerCCallableExportedName()
-                   << "', PACKAGE = '" << package() << "')"
+            ostr() << "    .Call(`" << registerCCallableExportedName()
+                   << "`, PACKAGE = '" << package() << "')"
```

Comment:
This may need a refinement to condition on whether registration is use or not just as the preceeding function does.

Reply:
And done in https://github.com/RcppCore/Rcpp/pull/1256/commits/f688928567568e73e1d84dc6f5c9956570d7b23b